lib/sms.rb

```diff
@@ -0,0 +1,21 @@
+require_relative "customer" # Not necessary except for the block below the class
+
+class Sms
+  attr_accessor :customer
+  attr_accessor :message
+
+  def initialize(customer)
+    @customer = customer
+    current_time = Time.now
+    deadline = current_time + 3600
+    @message = "Thank you! Your order was placed and will be delivered before #{deadline.strftime("%H:%M %p")}"
+  end
+end
+
+# This is not necessary, I was just playing around with it
+if __FILE__ == $0
```

Comment:
Yes.  Speaking of feature tests, I know I am a slow learner sometimes, but I am persistent, but even after spending a number of hours researching RSpec doubles/mocks/stubs, I don't know how to use them properly in my tests (actually AT ALL in my tests).